src/components/storage/InstallationScenario.jsx

```diff
@@ -73,60 +65,15 @@ const InstallationScenarioSelector = ({
     setIsFormValid,
     showStorage,
 }) => {
-    const { appliedPartitioning, diskSelection, partitioning } = useContext(StorageContext);
-    const { devices, mountPoints } = useOriginalDeviceTree();
+    const { diskSelection } = useContext(StorageContext);
+    const { mountPoints } = useOriginalDeviceTree();
     const selectedDisks = diskSelection.selectedDisks;
-    const [scenarioAvailability, setScenarioAvailability] = useState(Object.fromEntries(
-        scenarios.map((s) => [s.id, new AvailabilityState()])
-    ));
-    const diskTotalSpace = useDiskTotalSpace({ devices, selectedDisks });
-    const diskFreeSpace = useDiskFreeSpace({ devices, selectedDisks });
-    const mountPointConstraints = useMountPointConstraints({ devices, selectedDisks });
-    const usablePartitions = useUsablePartitions({ devices, selectedDisks });
-    const requiredSize = useRequiredSize();
     const { storageScenarioId } = useContext(StorageContext);
-    const originalExistingSystems = useOriginalExistingSystems();
-    const { defaultScheme } = useContext(StorageDefaultsContext);
-
-    useEffect(() => {
-        if ([diskTotalSpace, diskFreeSpace, mountPointConstraints, requiredSize, usablePartitions].some(itm => itm === undefined)) {
```

Comment:
Some of this checks got into `useAvailabilityMountPointMapping` hook:
```
        if ([usablePartitions, mountPointConstraints].some(data => data === undefined)) {
            return;
        }
```
in the PR. Other scenario hooks don't seem to have any. Not sure, maybe there need not be any but rather asking to be sure.

Reply:
I added all relevant checks in all scenario availability hooks. Thanks for catching this. :) 

For the record this was only working because JS will allow comparison of undefined with number.

``` js
>> undefined > 0
<- false
>> undefined < 0
<- false
```